### PR TITLE
Draft: Vcr tests continuation

### PR DESCRIPTION
**Pull request by jezekra1**

### CODE REVIEW

File: tests/features/models/cassettes/test_model_service/TestModelService.test_retrieve_model.yaml [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+      tokens","default":false,"nullable":true}},"additionalProperties":false}},"additionalProperties":false}}}}'
+    headers:
+      Connection:
+      - keep-alive
+      Date:
+      - Wed, 27 Dec 2023 15:39:54 GMT
+      Keep-Alive:
+      - timeout=72
+      Set-Cookie:
+      - BIGipServer~RIS3-INT-OCP-DAL12~Shared~ingress_fmaas_dev_fmaas_api=792414730.893.0000;
```
Comment (Tomas2D) [inline tests/features/models/cassettes/test_model_service/TestModelService.test_retrieve_model.yaml:125]: Remove

File: tests/features/prompt/cassettes/test_prompt_service/TestPromptService.test_create_delete_prompt.yaml [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+    content: '{"result":{"id":"7sUhNwy2VCQMQCJX","name":"My test prompt","data":{"meal":"goulash","author":"Shakespeare"},"input":"Write
+      the recipe for {{meal}} in the style of {{author}}","model_id":"google/flan-t5-xl","parameters":null,"moderations":{},"created_at":"2023-12-27T15:39:56.000Z","type":"private","public":false,"metadata":{"keywords":["meal","author"]},"author":{"id":924,"first_name":"Radek","last_name":"Jezek"},"messages":null}}'
+    headers:
+      Connection:
+      - keep-alive
+      Date:
+      - Wed, 27 Dec 2023 15:39:55 GMT
+      Keep-Alive:
+      - timeout=72
+      Set-Cookie:
```
Comment (Tomas2D) [inline tests/features/prompt/cassettes/test_prompt_service/TestPromptService.test_create_delete_prompt.yaml:29]: Remove

File: tests/features/prompt/cassettes/test_prompt_service/TestPromptService.test_create_delete_prompt.yaml [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+  response:
+    content: ''
+    headers:
+      Connection:
+      - keep-alive
+      Date:
+      - Wed, 27 Dec 2023 15:39:59 GMT
+      Keep-Alive:
+      - timeout=72
+      Set-Cookie:
```
Comment (Tomas2D) [inline tests/features/prompt/cassettes/test_prompt_service/TestPromptService.test_create_delete_prompt.yaml:149]: Remove

**Review by Tomas2D — CHANGES_REQUESTED** [review]

File: src/genai/features/text/generation/_generation_utils.py [hunk 4]
```diff
@@ -22,8 +26,17 @@ def generation_stream_handler(
             raise err
 
 
-def prompts_to_strings(prompts: Union[list[str], str]) -> list[str]:
+def prompts_to_strings(
```
[the remark this answers is not in the document]
Comment (jezekra1) [reply src/genai/features/text/generation/_generation_utils.py:29]: damn, forgot about this, thanks!

File: src/genai/features/text/chat/chat_generation_service.py [hunk 5]
```diff
@@ -48,7 +51,6 @@ def __init__(
         if not services:
             services = BaseServices()
 
-        self._limit = services.LimitService(api_client=api_client)
```
[the remark this answers is not in the document]
Comment (jezekra1) [reply src/genai/features/text/chat/chat_generation_service.py:51]: Do we want this limit also in the `ChatService`?
It wasn't used anywhere and chat doesn't have the async execution option.

[the remark this answers is not in the document]
Comment (Tomas2D) [reply src/genai/features/text/chat/chat_generation_service.py:51]: Ohh, I see; keep it as is (removed).

**Review by Tomas2D — APPROVED** [review]

**Pull request merged**